examples/user_guide/Param.ipynb

```diff
@@ -382,6 +398,29 @@
    "source": [
     "In this user guide we have seen how to leverage ``param`` to declare parameters, which ``panel`` can turn into a GUI with no overhead. Additionally we have seen how to link parameters to views and other parameters using the ``param.depends`` operator. This approach allows building complex and reactive ``panels``. In the [next user guide](Pipelines.ipynb) we will discover how to link multiple such classes into pipelines, making it possible to encapsulate complex workflows in clean self-contained classes."
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!/bin/bash\n",
+    "jq --indent 1 \\\n",
+    "    '\n",
+    "    (.cells[] | select(has(\"outputs\")) | .outputs) = []\n",
+    "    | (.cells[] | select(has(\"execution_count\")) | .execution_count) = null\n",
+    "    | .metadata = {\"language_info\": {\"name\":\"python\", \"pygments_lexer\": \"ipython3\"}}\n",
+    "    | .cells[].metadata = {}\n",
+    "    ' $1 > ~/Desktop/Param.ipynb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
```

Comment:
Looking good now, except for this.

Reply:
I don't see this in the current commit. I am assuming, this got fixed somehow